python-backend/app/api/now_applications/models/activity_summary/water_supply.py

```diff
@@ -9,18 +9,12 @@
 
 
 class WaterSupply(ActivitySummaryBase):
-    __tablename__ = "water_supply"
     __mapper_args__ = {
         'polymorphic_identity': 'water_supply',  ## type code
     }
 
-    activity_summary_id = db.Column(
-        db.Integer, db.ForeignKey('activity.activity_summary_id'), primary_key=True)
-
-    water_use_description = db.Column(db.String)
-    estimate_rate = db.Column(db.Numeric(14, 2))
-    pump_size = db.Column(db.Numeric(14, 2))
-    intake_location = db.Column(db.String)
+    ## NO TABLE FOR THIS TYPE
```

Comment:
Which activity types do not exist as tables? Are these comments still accurate?

Reply:
This comment is accurate. There is a water supply specific table at the detail level, but not at the summary level.